README.md

```diff
@@ -215,7 +218,7 @@ If you are coming from [jtarchie/github-pullrequest-resource][original-resource]
   - `label`
   - `git_config`: You can now get the pr/author info from .git/resource/metadata.json instead
 - `get`:
-  - `git.*`
+  - `git.*` (with the exception of `git.depth`, see above)
```

Comment:
haha, I've checked like 6 times and still missed it

Reply:
ah, no wait. I've intentionally left `git.depth` since we're talking about the parameters of the older resource that didn't make it. `git.depth` made it, but with a different name.

Does it make sense?

Reply:
Ah, my bad haha😆